setup.py

```diff
@@ -38,7 +38,6 @@ def run_tests(self):
         'django_nose',
         'mock',
         'coverage',
-        'mako',
```

Comment:
Seems like we need to keep this because xblock-utils needs it (per broken travis build logs)